modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro

```diff
@@ -662,7 +662,12 @@ definition {
 			key_picklistOption = ${newPicklistOption},
 			locator1 = "ObjectAdmin#ENTRY_PICKLIST_OPTION");
 
-		PortletEntry.save();
+		if (${invalidEntry} == "true") {
+			Button.clickSave();
+		}
+		else {
+			PortletEntry.save();
+		}
```

Comment:
The idea for this is good, but we're trying to move away from having the step to save within the macro because it could cause the macro to be inflexible. Update this macro to remove the save steps and just include whichever appropriate macro for saving you need on the testcase level.